committee-steering/governance/README.md

```diff
@@ -0,0 +1,34 @@
+# SIG Governance Template
+
+## Goals
+
+The following documents outline recommendations and requirements for SIG governance structure and provide
+template documents for SIGs to adapt.  The goals are to define the baseline needs for SIGs to self govern
+and organize in a way that addresses the needs of the core Kubernetes project.
+
+The documents are focused on:
+
+- Outlining organizational responsibilities
+- Outlining organizational roles
+- Outlining processes and tools
+
+Specific attention has been given to:
+
+- The role of technical leadership
+- The role of operational leadership
+- Process for agreeing upon technical decisions
+- Process for ensuring technical assets remain healthy
+
+## How to use the templates
+
+When developing or modifying a SIG governance doc, the intention is for SIGs to use the templates (*forthcoming*) as a [buffet]
+to choose options from.  It is recommended that SIGs start by looking at the
+[Recommendations and requirements] for SIG governance docs before selecting and defining strategies.
+
+The expectation is that SIGs will adapt the provided templates as best to meet the needs of the both the SIG
+and project.
+
+- [Recommendations and requirements]
+
+[Recommendations and requirements]: sig-governance-requirements.md
```

Comment:
I guess the URL's need to be in () here.

Reply:
This is a shortcut:

https://sourceforge.net/p/aliascommand/home/markdown_syntax/#md_ex_reflinks